paychmgr/paych.go

```diff
@@ -238,22 +197,22 @@ func (pm *Manager) CheckVoucherSpendable(ctx context.Context, ch address.Address
 	return true, nil
 }
 
-func (pm *Manager) getPaychRecipient(ctx context.Context, ch address.Address) (address.Address, error) {
+func (ca *channelAccessor) getPaychRecipient(ctx context.Context, ch address.Address) (address.Address, error) {
 	var state paych.State
-	if _, err := pm.sm.LoadActorState(ctx, ch, &state, nil); err != nil {
+	if _, err := ca.sm.LoadActorState(ctx, ch, &state, nil); err != nil {
 		return address.Address{}, err
 	}
 
 	return state.To, nil
 }
 
-func (pm *Manager) AddVoucher(ctx context.Context, ch address.Address, sv *paych.SignedVoucher, proof []byte, minDelta types.BigInt) (types.BigInt, error) {
-	pm.store.lk.Lock()
-	defer pm.store.lk.Unlock()
+func (ca *channelAccessor) addVoucher(ctx context.Context, ch address.Address, sv *paych.SignedVoucher, proof []byte, minDelta types.BigInt) (types.BigInt, error) {
+	ca.lk.Lock()
+	defer ca.lk.Unlock()
 
-	ci, err := pm.store.getChannelInfo(ch)
+	ci, err := ca.store.ByAddress(ch)
 	if err != nil {
-		return types.NewInt(0), err
+		return types.BigInt{}, err
```

Comment:
Does Lotus still have a BigInt type of its own? I feel like at this point all our money types should come from `spec-actors` abi package? Future refactor.

Reply:
It's an alias of the spec type, we just never got around to getting rid of it